nailed-it/New Group/PolishLibraryViewController.swift

```diff
@@ -149,3 +173,59 @@ extension UIImage {
         return img!
     }
 }
+
+extension PolishLibraryViewController: CZPickerViewDelegate, CZPickerViewDataSource {
+    func czpickerViewWillDisplay(_ pickerView: CZPickerView!) {
+        pickerView.setSelectedRows(self.selectedRows)
+    }
+
+    func numberOfRows(in pickerView: CZPickerView!) -> Int {
+        return self.brands.count
+    }
+
+    func numberOfRowsInPickerView(pickerView: CZPickerView!) -> Int {
+        return self.brands.count
+    }
+
+    func czpickerView(_ pickerView: CZPickerView!, titleForRow row: Int) -> String! {
+        return self.brands[row]
+    }
+
+    func czpickerView(_ pickerView: CZPickerView!, didConfirmWithItemAtRow row: Int){
+        self.navigationController?.setNavigationBarHidden(true, animated: true)
+    }
+
+    func czpickerViewDidClickCancelButton(_ pickerView: CZPickerView!) {
+        self.navigationController?.setNavigationBarHidden(false, animated: true)
+    }
+
+    func czpickerView(_ pickerView: CZPickerView!, didConfirmWithItemsAtRows rows: [Any]!) {
+        var selectedBrands: [String] = []
+        self.selectedRows = rows
+        var selectedBrandPolishes: [PolishColor] = []
+        for row in rows {
+            if let row = row as? Int {
+                selectedBrands.append(self.brands[row])
+                let brandQuery = PFQuery(className: "PolishColor")
+                for brand in selectedBrands {
+                    brandQuery.whereKey("brand", equalTo: brand)
+                }
+                brandQuery.order(byDescending: "brand")
+                brandQuery.findObjectsInBackground {
+                    (colors: [PFObject]?, error: Error?) -> Void in
+                    if error == nil {
+                        for color in colors! {
+                            var selectedBrandColor: PolishColor
```

Comment:
i don't think you need this var? can't you just do selectedBrandPolishes.append(color!)